src/web/lib/components/App/index.js

```diff
@@ -84,30 +105,18 @@ export const AppComponent = ({
             putTheme: storage.putTheme
           }}
         />
-        {loaderDelayExpired && <ThemeColorsEditor
-          {...{
-            theme,
-            selectedColor,
-            setColor,
-            setSelectedColor,
-            undo,
-            redo,
-            themeCanUndo,
-            themeCanRedo
-          }}
-        />}
-        {loaderDelayExpired && <ThemeBackgroundPicker {...{ theme, setBackground }} />}
+        </div>
       </BrowserPreview>
-      <ThemeUrl {...{ theme, urlEncodeTheme, clipboard }} />
       <PresetThemeSelector {...{ setTheme }} />
-      <SavedThemeSelector
+      {Object.keys(savedThemes).length > 0 && <SavedThemeSelector
```

Comment:
Only shows this if there are saved themes

Reply:
Nit: This is okay here, but might be cleaner as a selector in store.js - e.g. something like `haveStoredThemes()`. I left it out of selectors when I threw the count into the save button component earlier, so 🤷‍♂️ I could include it in a later PR for cleanups & refactorings